client/src/client.rs

```diff
@@ -267,12 +267,14 @@ impl ClientBuilder {
         let (pubkey, grpc) = match (&self.grpc_url, signer) {
             (Some(url), Some((pubkey, signer))) => {
                 #[cfg(not(target_arch = "wasm32"))]
-                let tx_client = TxClient::with_url(url, pubkey, signer).await?;
+                let builder = GrpcClientBuilder::with_url(url).with_native_roots();
```

Comment:
I think we should re-expose `tls-{native,webpki}-roots` feature flags in client and have `native` as default feature.

Can we make it in a way that if none of the `tls-xyz-roots` is selected we create here client without tls?